src/refinement-list/__tests__/refinement-list.spec.ts

```diff
@@ -105,21 +105,111 @@ describe('RefinementList', () => {
     });
   });
 
-  it('should call `toggleShowMore()` on click', () => {
-    const toggleShowMore = jest.fn();
-    const fixture = render({ toggleShowMore, canToggleShowMore: true });
-    fixture.componentInstance.testedWidget.limit = 5;
-    fixture.componentInstance.testedWidget.showMoreLimit = 10;
-    fixture.detectChanges();
+  describe('[show more]/[show less] button', () => {
+    const createFixture = (state, props) => {
+      const fixture = render(state);
+      Object.assign(fixture.componentInstance.testedWidget, props);
+      fixture.detectChanges();
+      return fixture;
+    };
+    it('should not show [show more] button when canToggleShowMore === false ', () => {
```

Comment:
The button should be visible when `canToggleShowMore` is `false` but with the disabled class (see [Vue](https://github.com/algolia/vue-instantsearch/blob/v2/src/components/RefinementList.vue#L76-L93)).

Reply:
ok that's another bug around the the showMore button.
https://github.com/algolia/angular-instantsearch/blob/bbe0e9631b4f09333097f5b01a7153889f031cb5/src/refinement-list/refinement-list.ts#L60

I understand now how that's not what `canToggleShowMore` was meant for. 
Wouldn't it be a breaking change to change this here? There may be users relying on this behaviour.

Reply:
Indeed, it can be solve separately then.

Reply:
ok, I filed an issue about this here to track it: https://github.com/algolia/angular-instantsearch/issues/391
PR is coming!

Reply:
I simplified the tests and in particular removed those specifying the inconsistent display of showmore button base on canToggleShowMore boolean.
I reintroduced them here where https://github.com/algolia/angular-instantsearch/pull/381/files with the correct behaviour